blocks/init/src/blocks/wrapper/components/wrapper-responsive-tab-content-simple.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { RangeControl, ToggleControl, Icon } from '@wordpress/components';
+import { icons } from '@eightshift/frontend-libs/scripts/editor';
+import globalSettings from '../../manifest.json';
+
+export const wrapperSpacingOptions = {
+  min: -10,
+  max: globalSettings.globalVariables.sectionSpacing.max,
+  step: globalSettings.globalVariables.sectionSpacing.step,
+};
+
+export const WrapperResponsiveTabContentSimple = (props) => {
+  const {
+    type,
+    spacingTop,
+    spacingBottom,
+    hideBlock,
+    onChangeSpacingTop,
+    onChangeSpacingBottom,
+    onChangeHideBlock,
+  } = props;
+
+  return (
+    <Fragment>
+      {onChangeSpacingTop &&
+        <RangeControl
+          label={
+            <Fragment>
+              <Icon icon={() => icons.spacingTop} />
```

Comment:
Why do we need to pass the icon name as an arrow function? Can this be extracted above the `return`?

Reply:
fixed, it was left from the previous implementation